src/amo/components/OverallRating.js

```diff
@@ -0,0 +1,42 @@
+import React, { PropTypes } from 'react';
+
+import translate from 'core/i18n/translate';
+
+import 'amo/css/OverallRating.scss';
+
+
+export class OverallRatingBase extends React.Component {
+  static propTypes = {
+    i18n: PropTypes.object,
+    addonName: PropTypes.string,
+  }
+
+  render() {
+    const { i18n, addonName } = this.props;
+    const prompt = i18n.sprintf(
+      i18n.gettext('How are you enjoying your experience with %(addonName)s?'),
+      { addonName });
+
+    return (
+      <div className="OverallRating">
+        <p>{prompt}</p>
+        <form action="">
+          <input id="love-it" value="love-it" name="overall-rating" type="radio" />
+          <label htmlFor="love-it" className="love-it">
+            {i18n.gettext('Love it!')}
+          </label>
+          <input id="it-is-ok" value="it-is-ok" name="overall-rating" type="radio" />
```

Comment:
I couldn't figure out a way to wrap the input with a background image and also hide the form control graphic. It seems like wrapping the label is common practice and I think it is semantic enough since, in this case, the real label (e.g. "I love it!") is a major piece of the UI